gobblin-kubernetes/gobblin-service/basic-cluster/application.yaml

```diff
@@ -0,0 +1,80 @@
+# In the future, build the kubernetes cluster from the official docker account
+# Also ensure that proper tagging/versioning is done i.e. remove :latest tag and instead use the digest of the container
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: 'gaas-app'
+  labels:
+    app: gaas-app
+spec:
+  selector:
+    matchLabels:
+      app: gaas-app
+  replicas: 1
+  template:
+    metadata:
+      name: 'gaas-app'
```

Comment:
Theres no collision because in kubernetes they differentiate objects of different types. Though I agree its going to get confusing so I'll rename the deployment to `gaas-deployment` and `gobblin-standalone-deployment`.